### ISSUE TITLE
Android does not build on react-native 0.65+

### ISSUE THREAD

**Issue by LadislavBohm** [transistorsoft/react-native-background-geolocation#1376]
## Your Environment
* Plugin version: 4.1.1
* Platform: Android
* OS version: any
* Device manufacturer / model: any
* React Native version (`react-native -v`): 0.65.1

## Expected Behavior
Should build

## Actual Behavior
Does not build because of outdated jcenter dependency that is not longer available.

## Steps to Reproduce
1. create new react native project (0.65.x) with this plugin
2. try to build Android version

## Context
I already fixed it myself locally and will create PR shortly.

**Comment by fahadbraveknight** [comment]
Thanks for the fix!

**Comment by christocracy** [comment]
This has already been updated in the private version, due for merge here soon.

I [sent the author of `android-permissions` $300 to expedite migration to *androidCentral*](https://github.com/nishkarsh/android-permissions/issues/20#issuecomment-883497520) back in July.

**Comment by christocracy** [comment]
One can also fix this in their `android/build.gradle` by adding `jCenter` to repositories.  While `jCenter` is now deprecated for *new* releases, it will be available read-only *indefinitely*.

```gradle
repositories {
    google()
    jcenter()  // <-------------- add this
    mavenCentral()
}
```

**Comment by LadislavBohm** [comment]
Closed by #1381
